caravel/templates/caravel/import_dashboards.html

```diff
@@ -0,0 +1,13 @@
+{% extends "caravel/basic.html" %}
+{% block title %}{{ _("Import") }}{% endblock %}
+{% block body %}
+    {% include "caravel/flash_wrapper.html" %}
+    <div class="container">
+        <title>Import the dashboards.</title>
+        <h1>Import the dashboards.</h1>
+        <form action="" method=post enctype=multipart/form-data>
```

Comment:
since `action=""` you can leave it out altogether and it will still post to the current url.